lib/doorkeeper/models/concerns/secret_storable.rb

```diff
@@ -66,6 +66,8 @@ def find_by_fallback_token(attr, plain_secret)
           # Use the previous strategy to look up
           stored_token = fallback_secret_strategy.transform_secret(plain_secret)
           find_by(attr => stored_token).tap do |resource|
```

Comment:
Why not `return` instead of break?

Reply:
![image](https://user-images.githubusercontent.com/12478720/55194271-87a5d780-517f-11e9-92a1-2eeda9928b07.png)

@nbulaj Got this lint warning from rubocop. I'll gladly change it to `return`, which is what I wrote in the first place :)